assets/xml/overlays/ovl_En_Sth.xml

```diff
@@ -1,14 +1,14 @@
 <Root>
     <File Name="ovl_En_Sth" BaseAddress="0x80B66D30" RangeStart="0x1890" RangeEnd="0x646C">
-        <DList Name="ovl_En_Sth_DL_0034A0" Offset="0x34A0"/>
-        <DList Name="ovl_En_Sth_DL_0037C0" Offset="0x37C0"/>
-        <Animation Name="ovl_En_Sth_Anim_003F50" Offset="0x3F50" /> <!-- Original name is "sth_ko_matsu" ("matsu" = "to wait​") -->
-        <Animation Name="ovl_En_Sth_Anim_0045B4" Offset="0x45B4" /> <!-- Original name is "sth_kibunwaru" ("bad mood") -->
-        <Animation Name="ovl_En_Sth_Anim_004CC0" Offset="0x4CC0" /> <!-- Original name is "sth_talk2000" -->
-        <Animation Name="ovl_En_Sth_Anim_00533C" Offset="0x533C" /> <!-- Original name is "sth_wait2000" -->
-        <Animation Name="ovl_En_Sth_Anim_0059B8" Offset="0x59B8" /> <!-- Original name is "sth_miageru" ("look up") -->
-        <Animation Name="ovl_En_Sth_Anim_005E40" Offset="0x5E40" /> <!-- Original name is "sth_kyorokyoro" (onomatopoeic for "looking around restlessly") -->
-        <Animation Name="ovl_En_Sth_Anim_006354" Offset="0x6354" /> <!-- Original name is "sth_onegai" ("request; favor (to ask); wish") -->
-        <Animation Name="ovl_En_Sth_Anim_00645C" Offset="0x645C" /> <!-- Original name is "sth_tasukete" ("help!​") -->
+        <DList Name="gEnSthHeadDL" Offset="0x34A0"/>
+        <DList Name="gEnSthExtraHairDL" Offset="0x37C0"/>
+        <Animation Name="gEnSthSignalingAnim" Offset="0x3F50" /> <!-- Original name is "sth_ko_matsu" ("matsu" = "to wait​") -->
+        <Animation Name="gEnSthBendingDownAnim" Offset="0x45B4" /> <!-- Original name is "sth_kibunwaru" ("bad mood") -->
+        <Animation Name="gEnSthTalkingWithHandUpAnim" Offset="0x4CC0" /> <!-- Original name is "sth_talk2000" -->
+        <Animation Name="gEnSthWaitingAnim" Offset="0x533C" /> <!-- Original name is "sth_wait2000" -->
+        <Animation Name="gEnSthLookUpAnim" Offset="0x59B8" /> <!-- Original name is "sth_miageru" ("look up") -->
+        <Animation Name="gEnSthLookingAroundRestlesslyAnim" Offset="0x5E40" /> <!-- Original name is "sth_kyorokyoro" (onomatopoeic for "looking around restlessly") -->
+        <Animation Name="gEnSthBeggingAnim" Offset="0x6354" /> <!-- Original name is "sth_onegai" ("request; favor (to ask); wish") -->
+        <Animation Name="gEnSthPanicShakingAnim" Offset="0x645C" /> <!-- Original name is "sth_tasukete" ("help!​") -->
```

Comment:
Something that applies for most of these: I've been trying to avoid using gerunds for animation names, unless avoiding them makes the name super awkward. So for example, something like `gEnSthSignalAnim` instead of `gEnSthSignalingAnim`. What do you think? There's already `gEnSthLookUpAnim`, which is named without gerunds, so the rest would look like that.

Reply:
Oh right forgot that was the project preference. I'm fine with it, but my own preference is the other way, animations describe what they do in realtime, as an action, makes more sense to me, I just forgot so yeah I can change it.

Reply:
Yeah let me be clear; I don't think the "no gerunds" thing is a strong preference. It comes from when I was naming a lot of animations, and that was the feedback I got. I think it's probably fine to name these with gerunds if you want, but in that case, name it `gEnSthLookingUpAnim` so they're all consistent.